.travis.yml

```diff
@@ -1,3 +1,4 @@
+sudo: false
```

Comment:
Still we are using libssl https://github.com/travis-ci/travis-ci/issues/3976 and mango are running through super user.

``` YAML
addons:
  apt:
    packages:
      - openssl

services:
  - mongodb
```